src/channel.cpp

```diff
@@ -69,6 +69,7 @@ refCount(0) {
 
 Channel::~Channel() {
     assert(participants.size() == 0);
```

Comment:
What happens if one of these assertions fails?

Reply:
All right, if you've confirmed that these only run in debug mode, then it's good.